src/types/settings.ts

```diff
@@ -1,7 +1,9 @@
 export interface ISettingsProps {
   actions?: {
+    changeMode: Function;
     signOut: Function;
     changeTheme: Function;
+    togglePermissionsPanel: Function;
```

Comment:
Unrelated to task: `togglePermissionsPanel` was the only action creator in `Settings.tsx` not listed here, so I added it for consistency. However, I don't think the `actions` property is actually used anywhere, so this change is purely for cleanliness purposes.

Reply:
yeah you're right, when i remove the actions nothing happens (it doesn't even rerender the site). i guess we need to ask the GE team about this. probably leave it as is with the added `togglePermissionsPanel` for completeness sake